### PR TITLE
Transfer and refine

### PR DESCRIPTION
**Pull request by TMcSquared**
@beeperdeeper089  @DontBelieveMe 
You guys need to test this and show me where to adjust as I don't know python all that well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Project
```
Comment (NewbProgrammer101) [inline README.md]: The title should be better than this.

Comment (beeperdeeper089) [reply README.md]: That isn't supposed to be a title, that's literally a heading saying "Project", with the description of the project below it.

Unless I'm interpreting wrong :worried:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+This is the Development Utilities for the Client.
```
Comment (beeperdeeper089) [inline README.md]: Use the NEW readme "design" in like the Engine repo and such. Edit it so it works. Don't use this, makes it too objective and not "inclusive".

**Comment by beeperdeeper089** [comment]
I'm going to close this PR, as it's become old/outdated now.

**Comment by TMcSquared** [comment]
kk

**Pull request closed without merging**